packages/gen-ai-hub/src/orchestration/orchestration-filter-utility.test.ts

```diff
@@ -0,0 +1,175 @@
+      constructCompletionPostRequest(genaihubCompletionParameters);
+    expect(
+      completionPostRequest.orchestration_config.module_configurations
+        .filtering_module_config
+    ).toEqual(expectedFilterConfig);
+  });
+  it('constructs filter configuration with both input and ouput', async () => {
+    const filterConfig: FilteringModuleConfig = {
+      input: azureContentFilter({
+        Hate: 4,
+        SelfHarm: 0,
+        Sexual: 2,
+        Violence: 6
+      }),
+      output: azureContentFilter({ Sexual: 2, Violence: 6 })
+    };
+    const expectedFilterConfig: FilteringModuleConfig = {
+      input: {
+        filters: [
+          {
+            type: 'azure_content_safety',
+            config: {
+              Hate: 4,
+              SelfHarm: 0,
+              Sexual: 2,
+              Violence: 6
+            }
+          }
+        ]
+      },
+      output: {
+        filters: [
+          {
+            type: 'azure_content_safety',
+            config: {
+              Sexual: 2,
+              Violence: 6
+            }
+          }
+        ]
+      }
+    };
+    genaihubCompletionParameters.filterConfig = filterConfig;
+    const completionPostRequest: CompletionPostRequest =
+      constructCompletionPostRequest(genaihubCompletionParameters);
+    expect(
+      completionPostRequest.orchestration_config.module_configurations
+        .filtering_module_config
+    ).toEqual(expectedFilterConfig);
+  });
+
+  it('use defaults for filter configuration', async () => {
+    const filterConfig: FilteringModuleConfig = {
+      input: azureContentFilter(),
+      output: azureContentFilter()
+    };
+    genaihubCompletionParameters.filterConfig = filterConfig;
+    const completionPostRequest: CompletionPostRequest =
+      constructCompletionPostRequest(genaihubCompletionParameters);
+    const expectedFilterConfig: FilteringModuleConfig = {
+      input: {
+        filters: [
+          {
+            type: 'azure_content_safety'
+          }
+        ]
+      },
+      output: {
+        filters: [
+          {
+            type: 'azure_content_safety'
+          }
+        ]
+      }
+    };
+    expect(
+      completionPostRequest.orchestration_config.module_configurations
+        .filtering_module_config
+    ).toEqual(expectedFilterConfig);
+  });
+
+  it('skips filter configuration', async () => {
+    const filterConfig: FilteringModuleConfig = {};
+    genaihubCompletionParameters.filterConfig = filterConfig;
+    const completionPostRequest: CompletionPostRequest =
+      constructCompletionPostRequest(genaihubCompletionParameters);
+    expect(
+      completionPostRequest.orchestration_config.module_configurations
+        .filtering_module_config
+    ).toBeUndefined();
+  });
+
+  it('throw error when configuring empty filter', async () => {
+    const createFilterConfig = () => {
+      {
+        azureContentFilter({});
+      }
+    };
+    expect(createFilterConfig).toThrow;
```

Comment:
[req] 1. You need to call the `toThrow()` method otherwise it will not be executed correctly. I believe `expect(createFilterConfig).not.toThrow;` should have the same effect.
2. Consider also checking that the correct error message is thrown.
```suggestion
    expect(createFilterConfig).toThrow();
```